tests/Resources/Page/LoginPage.robot

```diff
@@ -48,3 +48,17 @@ Login To Openshift
     Input Text  id=inputPassword  ${ocp_user_pw}
     Click Element  xpath=/html/body/div/div/main/div/form/div[4]/button
     Maybe Skip Tour
+
+Log In Should Be Requested
+    [Documentation]    Passes if the login page appears and fails otherwise
+    ${present}=    Is OpenShift Login Visible
+    IF    ${present} == ${FALSE}
+        Fail    msg=Log in page did not appear as expected       
+    END
```

Comment:
I would do an inline IF but it's fine either way

Reply:
done